<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visibleTabs</class>
 <widget class="QDialog" name="visibleTabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup visible tabs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Visible tab pages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkDisk">
     <property name="text">
      <string>Disks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkRaid">
     <property name="text">
      <string>Raid</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkFsTab">
     <property name="text">
      <string>FSTab</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkIsci">
     <property name="text">
      <string>Iscsi</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkStats">
     <property name="text">
      <string>Stats</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkLVM">
     <property name="text">
      <string>LVM</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
